.travis/run.sh

```diff
@@ -13,7 +13,6 @@ coverage run \
     -m py.test \
     -Wd \
     --travis-fold=always \
-    -vvvvvv \
```

Comment:
Should we leave some verbosity?

Reply:
I think it'd be sufficient to just remove geth or py-evm logs which are quite verbose. But personally I almost never read Travis logs output so I don't care.

Reply:
it's the next line `--log-config='raiden:DEBUG' ` it enables debug logging for raiden.

this is disabling the `DEBUG` level for everything that is not raiden [1](https://github.com/raiden-network/raiden/blob/master/raiden/tests/conftest.py#L152-L153), and the [geth verbosity](https://github.com/raiden-network/raiden/blob/master/raiden/tests/utils/blockchain.py#L94)